dist/privacy.js

```diff
@@ -296,5 +296,84 @@ return JSON.stringify({
     navigator_geolocation_watchPosition: testPropertyStringInResponseBodies(
       'navigator.+geolocation.+watchPosition'
     ),
-  }
+  },
+  
+  fingerprinting2024: (() => {
```

Comment:
Can we remove the 2024 from this? Unless we're planning to have a different list every year and compare them, it seems redundant.

```suggestion
  fingerprinting: (() => {
```